src/set.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+export default class Set {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  add(value) {
+    if (this.elements.indexOf(value) === -1) {
+      this.elements[this.elements.length] = value
+      return this.elements
```

Comment:
`this.elements` is for internal use only and should never be returned. I think you want `return true`